packages/notifications/src/useNotifications.ts

```diff
@@ -0,0 +1,36 @@
+import { createContext, createElement, PropsWithChildren, useContext } from 'react';
+
+export type NotificationType = 'error' | 'warn' | 'info' | 'success' | 'welcome';
```

Comment:
Consider not promoting `welcome` to the package? It's used in one (niche) place, where I suspect it would be fine to use `info` instead.

```suggestion
export type NotificationType = 'error' | 'warn' | 'info' | 'success';
```

Reply:
hmmm... I debated this myself, but "welcome" seems a common enough thing to want... and while the logical difference to info is minimal, it is a11y related with a much more helpful "welcome notification" aria label being used... so I am loathe to remove that seeing as how it is not a huge impact on the types to include it.

Reply:
Fair enough.

Though I would claim that it's single use in Terra UI, for a [notification shown when redirected from the old Firecloud](https://github.com/DataBiosphere/terra-ui/blob/5aa6b29bc6b86b36f2742079f45fb6639ae3148b/src/components/FirecloudNotification.js), suggests that it's not something frequently needed.